content/docs/concepts/security/index.md

```diff
@@ -331,11 +334,11 @@ spec:
     methods: ["GET", "HEAD"]
 {{< /text >}}
 
-In addition, we support **prefix matching** and **suffix matching** for all the fields in a rule. For example, you can define a "tester" role that
-has the following permissions in "default" namespace:
-* Full access to all services with prefix "test-" (e.g, "test-bookstore", "test-performance", "test-api.default.svc.cluster.local").
-* Read ("GET") access to all paths with "/reviews" suffix (e.g, "/books/reviews", "/events/booksale/reviews", "/reviews")
-in service "bookstore.default.svc.cluster.local".
+In addition, we support prefix matching and suffix matching for all the fields in a rule. For example, you can define a “tester” role
```

Comment:
Replace:

In addition, you can use prefix and suffix matching for all fields in a rule.

Reply:
Done.